### DISCUSSION TITLE
openebs cstor csi driver: volumeMode: Block (PVC) is not mounting to pod

### DISCUSSION THREAD

**Discussion by xigifty** [openebs/openebs#3483] [General] [answered]
Hi All, need some quick help, as teams are waiting on me. 

I have deployed the cstor openebs type and able to follow all the steps in the read me and everything is working fine. I was able to create snapshots, restore snapshots, a bunch of tasks. 
https://github.com/openebs/cstor-operators/blob/develop/docs/quick.md

https://github.com/openebs/cstor-operators/blob/develop/docs/tutorial/volumes/block_volume.md

I was able to create pvc with volumeMode: Filesystem and once i mounted to the pod i can see it attached to the pod. 

But however i create a pvc with volumeMode: Block and once i mounted to the pod its "NOT" attached to the pod, The pods logs are clearly fine and its not showing any errors. 

Please help.

**Comment by avishnu** [comment, marked as answer]
Apologies for the super late response. Assuming that you'd have moved on with your deployment, since it's been more than a couple of years !!
Having said that, we sincerely recommend you to try Mayastor, which is the latest high-performant replicated storage engine and replacement for CStor/Jiva.
